core/standard/clause_7_core.adoc

```diff
@@ -251,14 +251,9 @@ Servers implementing <<cross_origin,CORS>> will implement the method OPTIONS, to
 [[http_status_codes]]
 ==== HTTP status codes
 
-<<status_codes>> lists the main HTTP status codes that clients should be
-prepared to receive.
+This API standard does not impose any restictions on which features of the HTTP and HTTPS protocols may be used. API clients should be prepared to handle any legal HTTP or HTTPS status code. 
```

Comment:
Maybe add something like: "Note that clients may receive responses with status codes that have not been created by the API and that are not described in the API definition. For example, a response with status code 504 created by a proxy."